static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -73,6 +73,14 @@ Sentry.init({
   // Here, we'll capture profiles for 100% of transactions.
   profilesSampleRate: 1.0,`
       : ''
+  }${
+    params.isReplaySelected
+      ? `
+  // Record Session Replays for 10% of Sessions and 100% of Errors
```

Comment:
RN will have the `enableExperimentalViewRenderer` option soon ([PR in review](https://github.com/getsentry/sentry-react-native/pull/4660)), same considerations as for iOS (see  https://github.com/getsentry/sentry/pull/87304/files#r2001723725) apply